resources/views/platform/components/developer/inspector-toolbox.blade.php

```diff
@@ -0,0 +1,167 @@
+@props([
+    'canHaveTypes' => true,
+    'gameId' => 0,
+    'modificationLevel' => 'none', // 'none' | 'partial' | 'full'
+    'isManagingCoreAchievements' => true,
+])
+
+<?php
+use App\Platform\Enums\AchievementFlag;
+use App\Platform\Enums\AchievementType;
+
+$progressionType = AchievementType::Progression;
+$winConditionType = AchievementType::WinCondition;
+$officialFlag = AchievementFlag::OfficialCore;
+$unofficialFlag = AchievementFlag::Unofficial;
+?>
+
+@if ($modificationLevel !== 'none')
+    <script>
+    function toolboxComponent() {
+        return {
+            areCodeRowsHidden: true,
+
+            /**
+             * @param {'flag' | 'type'} property
+             * @param {3 | 5 | 'progression' | 'win_condition' | null} newValue
+             * @param {number} selectedCount
+             */
+            getConfirmMessage(property, newValue, selectedCount) {
```

Comment:
I see this in the page source, but I get the following console error trying to set the type of an achievement or promote/demote one:
```
app-d237b129.js:1 Alpine Expression Error: getConfirmMessage is not defined

Expression: "updateAchievementsProperty('type', 'progression')"

 <button class=​"btn" @click=​"updateAchievementsProperty('type', 'progression')​" fdprocessedid=​"bok2k2">​ Set Selected to Progression ​</button>​flex
Fe @ app-d237b129.js:1
(anonymous) @ app-d237b129.js:5
Promise.catch (async)
(anonymous) @ app-d237b129.js:5
mo @ app-d237b129.js:1
(anonymous) @ app-d237b129.js:5
o @ app-d237b129.js:5
(anonymous) @ app-d237b129.js:5
(anonymous) @ app-d237b129.js:5
app-d237b129.js:5 Uncaught ReferenceError: getConfirmMessage is not defined
    at Proxy.updateAchievementsProperty (achievementinspector.php?g=4381:1264:35)
    at eval (eval at <anonymous> (app-d237b129.js:5:640), <anonymous>:3:32)
    at app-d237b129.js:5:979
    at mo (app-d237b129.js:1:4883)
    at app-d237b129.js:5:35726
    at o (app-d237b129.js:5:25843)
    at app-d237b129.js:5:26832
    at HTMLButtonElement.<anonymous> (app-d237b129.js:5:25865)
```

Reply:
Accidentally broke this during a last-minute refactor. Resolved in latest.